### DISCUSSION TITLE
Word count (in addition to Reading Time)

### DISCUSSION THREAD

**Discussion by kris-brown** [quarto-dev/quarto-cli#6404] [Feature Requests]
### Description

Hello! Document listings support a `reading-time` field (which assumes a 200 word/min rate). Is it possible to have access to the estimated word count directly as something that could appear as a field in a listing? Or if there is some way of accessing the word count computed by https://github.com/andrewheiss/quarto-wordcount as a listing field, that would also work!

**Comment by mcanouil** [comment]
Closing as resolved.
See #6407
